construction/data/knowledge_article.xml

```diff
@@ -167,249 +358,427 @@
+            data-oe-role="status" contenteditable="false" role="status">
+            <i class="o_editor_banner_icon mb-3 fst-normal" data-oe-aria-label="Banner Info"
+                aria-label="Banner Info">🥇</i>
+            <div class="o_editor_banner_content o-contenteditable-true w-100 px-3"
+                contenteditable="true">
+                <p>
+                    <strong>Tips for success</strong>
+                </p>
+                <ul>
+                    <li>
+                        <div class="o-paragraph"><strong>Standardize templates:</strong> Use
+                            quotation and project templates to save time and maintain consistency.</div>
+                    </li>
+                    <li>
+                        <div class="o-paragraph"><strong>Keep documentation organized:</strong>
+                            Centralize contracts, drawings, and permits for easy access.</div>
+                    </li>
+                    <li>
+                        <div class="o-paragraph"><strong>Monitor stock proactively:</strong> Avoid
+                            project delays by tracking inventory in real-time.</div>
+                    </li>
+                    <li>
+                        <div class="o-paragraph"><strong>Track progress daily:</strong> Encourage
+                            teams to log time and update task status regularly.</div>
+                    </li>
+                    <li>
+                        <div class="o-paragraph"><strong>Use dashboards:</strong> Set up reporting
+                            dashboards to monitor costs, schedules, and client satisfaction.</div>
+                    </li>
+                </ul>
+            </div>
+        </div>
         <p>
-            Use the <strong><font class="text-o-color-1">Accounting</font></strong> app to easily handle:
+             <br/>
         </p>
+        <h2>Do You Want to Go Further?</h2>
+        <hr/>
+        <p>Explore additional apps to enhance your construction business:</p>
         <ul>
             <li>
-                The customer reminders.
+                <p><strong>Employees</strong> – Manage workforce, skills, and certifications.</p>
             </li>
             <li>
-                The vendor bills payment.
+                <p><strong>Fleet</strong> – Track machinery, vehicles, and maintenance schedules.</p>
             </li>
             <li>
-                The synchronisation with the banks.
+                <p><strong>Website</strong> – Publish projects, showcase services, and capture
+                    leads.</p>
             </li>
             <li>
-                The financial reporting.
+                <p><strong>Accounting</strong> – Advanced financial management and reporting.</p>
+            </li>
+            <li>
+                <p><strong>Sign</strong> – Collect digital signatures for contracts and approvals.</p>
```

Comment:
The manifest should be adapted, since sign, hr (and maybe others) are not meant to be in the dependency list anymore)